src/dummy_models.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+from sklearn.metrics import (
+    accuracy_score,
+    precision_score,
+    recall_score,
+    f1_score,
+)
+
+
+def dummy_always_bald(val_data):
+    return np.ones(len(val_data), dtype=int)
+
+
+def dummy_always_non_bald(val_data):
+    return np.zeros(len(val_data), dtype=int)
+
+
+def dummy_random(val_data):
+    return np.random.randint(2, size=len(val_data))
+
```

Comment:
I would add something like DummyClassifier(strategy="most_frequent") from sklearn.dummy

Reply:
changed to subclass of keras Model